### DISCUSSION TITLE
RaspAP not working with zerotier (and vice versa)

### DISCUSSION THREAD

**Discussion by AleXSR700** [RaspAP/raspap-webgui#1166] [Q&A]
Hello everyone,

I was running zerotier on my system before installing RaspAP. Now that I have RaspAP installed, zerotier is not working anymore.

At first dnsmasq would not start in RaspAP so I executed
`sudo systemctl stop systemd-resolved
sudo systemctl disable systemd-resolved
sudo systemctl mask systemd-resolved`

Then dnsmasq started in RaspAP.

But now zerotier is not working.
`sudo zerotier-cli info
Error connecting to the ZeroTier service: 

Please check that the service is running and that TCP port 9993 can be contacted via 127.0.0.1.
`
Does anybody know how to run both in parallal or whether this is even possible?

I am using wlan0 to connect to router and wlan1 as AP.

Thank you for your support
Alex

**Comment by billz** [comment]
Sounds like ZeroTier is incompatible. RaspAP uses dhcpcd to configure TCP/IP across all available network interfaces. dnsmasq is used for DHCP and DNS. If ZeroTier uses `systemd-networkd` it will conflict with this project. I've never used it but perhaps running it in a Docker container will work.

**Reply by johncarterofmars** [reply]
I can only say that it is running for me. Installed RaspAP first, got it all working, then installed ZTO and it works normally for me. If there is anything anyone would like me to check, I can. But I am using it as I speak to access my RaspAP.

**Reply by johncarterofmars** [reply]
By the way, what I ended up doing to allow it to work was this: 
iptables -t nat -I POSTROUTING -d 127.0.0.0/8 -j ACCEPT

**Comment by jsn0327** [comment]
> Hello everyone,
> 
> I was running zerotier on my system before installing RaspAP. Now that I have RaspAP installed, zerotier is not working anymore.
> 
> At first dnsmasq would not start in RaspAP so I executed `sudo systemctl stop systemd-resolved sudo systemctl disable systemd-resolved sudo systemctl mask systemd-resolved`
> 
> Then dnsmasq started in RaspAP.
> 
> But now zerotier is not working. `sudo zerotier-cli info Error connecting to the ZeroTier service:
> 
> Please check that the service is running and that TCP port 9993 can be contacted via 127.0.0.1. ` Does anybody know how to run both in parallal or whether this is even possible?
> 
> I am using wlan0 to connect to router and wlan1 as AP.
> 
> Thank you for your support Alex

I have found a fix for this.  See my post [Here](https://github.com/RaspAP/raspap-webgui/issues/1270)
